test/exportable/ExportableConfiguration.test.ts

```diff
@@ -64,5 +76,11 @@ describe('ExportableConfiguration', () => {
     it('should include the version when present', () => {
       expect(result).toMatch(/^version: 0\.13.0/m);
     });
+
+    it('should include the dependencies when present', () => {
+      expect(result).toMatch(/^dependencies:/m);
+      expect(result).toMatch(/hl7\.fhir\.us\.core: 3\.1\.0/m);
```

Comment:
As it is, I think this would match `foohl7.fhir.us.core: 3.1.0bar`.  I think you can be more specific by adding start/end anchors:
```
      expect(result).toMatch(/^\s+hl7\.fhir\.us\.core: 3\.1\.0$/m);
```
I didn't test it, but I *think* it should work?